repositories-list.md

```diff
@@ -26,6 +26,12 @@ Templates for making issues and pull requests.
 - [Virtual Coffee / podcast transcripts](https://github.com/Virtual-Coffee/podcast-transcripts)
 - [freeCodeCamp's Developer Quiz Site](https://github.com/freeCodeCamp/Developer_Quiz_Site)
 - [EddieHubCommunity / BioDrop](https://github.com/EddieHubCommunity/BioDrop)
+- [LinksHub](https://github.com/rupali-codes/LinksHub)
+- [Codecademy/docs](https://github.com/Codecademy/docs)
+- [TheOdinProject/curriculum](https://github.com/TheOdinProject/curriculum)
+- [MDN Web Docs/content](https://github.com/mdn/content)
+- [OpenSauced/docs](https://github.com/open-sauced/docs)
```

Comment:
Hey @CBID2,
I'm not sure if you have removed the OpenSauced Docs repo here. Because there is a duplication in line 35.

Reply:
I removed the duplicate @adiati98.